### DISCUSSION TITLE
Support HTML5 video

### DISCUSSION THREAD

**Discussion by dominictobias** [npm/feedback#665] [Registry] [answered]
Hey, GitHub has supported HTML5 videos in Markdown for a while now which is great, no more converting to inferior GIFs online.

We simply need to put in a video url and it will render:

https://user-images.githubusercontent.com/760314/162579030-e2268d06-e499-46db-ade3-f0a634a13c35.mov

It would be great to see NPM support this too, instead of seeing:

<img width="761" alt="Screenshot 2022-04-10 at 23 07 43" src="https://user-images.githubusercontent.com/760314/162641971-a2f97532-9e01-4c91-9ebf-9932a4d162b0.png">

**Comment by monishcm** [comment, marked as answer]
Unfortunately this is not a feature we are investing right now, but we will consider your feedback when we pick this up. Thanks for pitching in.
